build.gradle

```diff
@@ -18,10 +19,21 @@ archivesBaseName = 'cfr'
 
 sourceSets {
     main {
-        java.srcDirs = ['src']
+        java.srcDirs = ['src', 'src-templates']
     }
 }
 
+java {
+    withSourcesJar()
+}
+
+blossom {
+    replaceTokenIn 'src-templates/org/benf/cfr/reader/util/CfrVersionInfo.java'
+    replaceToken '${project.version}', project.version
```

Comment:
Ah nice, I should have looked into this. We should make it clear that this is a fork, possibly just append fabricmc/crf to the version metadata here?

Such as:

version + '+build.fabric-crf` or something idk.